frontend/src/app/components/EmailDialog.js

```diff
@@ -68,15 +73,21 @@ export default class EmailDialog extends React.Component {
     return (
       <div id="second-page" className="modal">
         <header className="modal-header-wrapper">
-          <h3 className="modal-header" data-l10n-id="emailOptInConfirmationTitle">Email Sent</h3>
+          <Localized id="emailOptInConfirmationTitle">
+            <h3 className="modal-header">Email Sent</h3>
+          </Localized>
           <div className="modal-cancel" onClick={e => this.continue(e)} />
         </header>
         <div className="modal-content centered">
           <div className="envelope" />
-          <p data-l10n-id="emailOptInSuccessMessage2">Thank you!</p>
+          <Localized id="emailOptInSuccessMessage2">
+            <p>Thank you!</p>
+          </Localized>
         </div>
         <div className="modal-actions">
-          <button id="email-success-continue" onClick={e => this.continue(e)} className="button default large" data-l10n-id="emailOptInConfirmationClose">On to the experiments...</button>
+          <Localized id="emailOptInConfirmationClose">
+            <button id="email-success-continue" onClick={e => this.continue(e)} className="button default large">On to the experiments...</button>
```

Comment:
I think @flodolo generally recommends the `&hellip;` "&hellip;" over the 3 separate dots "...".